shell/run_shell_cmd.go

```diff
@@ -22,5 +23,26 @@ func RunShellCommand(terragruntOptions *options.TerragruntOptions, command strin
 
 	cmd.Dir = terragruntOptions.WorkingDir
 
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, forwardSignals...)
+	defer signal.Stop(c)
+	defer close(c)
+
+	go func() {
+		for {
+			select {
+			case s := <-c:
+				if s == nil {
+					return
```

Comment:
Is this a normal case or indicate of an odd problem? Perhaps it should be logged?